scripts.py

```diff
@@ -519,86 +519,6 @@ def fix_identifier_with_equivalents(self, primary_identifier, equivalent_ids):
         print "I WOULD NOW EXPECT EVERYTHING TO BE FINE."
 
 
-class RedoOCLCForThreeMScript(Script):
```

Comment:
This script is irrelevant now (metadata wrangler doesn't even keep track of Bibliotheca identifiers anymore) and the tests were failing, so I got rid of the whole thing.